overlord/ifacestate/ifacestate_test.go

```diff
@@ -487,7 +487,7 @@ func (s *interfaceManagerSuite) TestAutoconnectDoesntConflictOnInstallingDiffere
 	c.Assert(ignore, Equals, false)
 	c.Assert(ifacestate.CheckAutoconnectConflicts(s.state, "consumer", "producer"), IsNil)
 
-	ts, err := ifacestate.ConnectPriv(s.state, "consumer", "plug", "producer", "slot", []string{"auto"})
+	ts, err := ifacestate.ConnectPriv(s.state, "consumer", "plug", "producer", "slot", ifacestate.ConnectOpts(true, false))
```

Comment:
When reading "ifacestate.ConnectsOpts(true, false)" its not entirely clear for the casual reader what true and false refers to here. I wonder if we can find something that makes this easier to read. I understand this is just a test-helper so its not a big deal.

Maybe in export_test: "NewConnectOptsWithAutoSet" and once we need something else in the tests we add "NewConnectOptsWithGadgetSet". Or "NewConnectOpts" and "ConnectOptsSetAuto", "ConnectOptsSetGadget".